src/MIPSolver/MIPSolverGurobi.cpp

```diff
@@ -392,10 +392,21 @@ void MIPSolverGurobi::initializeSolverSettings()
         // gurobiModel->getEnv().set(GRB_DoubleParam_IntFeasTol, 1e-6);
         // gurobiModel->getEnv().set(GRB_DoubleParam_OptimalityTol, 1e-6);
         // gurobiModel->getEnv().set(GRB_DoubleParam_MarkowitzTol, 1e-4);
-        // gurobiModel->getEnv().set(GRB_DoubleParam_NodeLimit, 1e15);
+
         gurobiModel->getEnv().set(GRB_IntParam_SolutionLimit, GRB_MAXINT);
         gurobiModel->getEnv().set(
             GRB_IntParam_SolutionNumber, env->settings->getSetting<int>("MIP.SolutionPool.Capacity", "Dual") + 1);
+
+        // Adds a user-provided node limit
+        if(env->settings->getSetting<double>("MIP.NodeLimit", "Dual") > 0)
+        {
+            gurobiModel->getEnv().set(
+                GRB_DoubleParam_NodeLimit, env->settings->getSetting<double>("MIP.NodeLimit", "Dual"));
+        }
+
+        if(env->settings->getSetting<double>("MIP.NodeLimit", "Dual") > 0)
+        {
+        }
```

Comment:
???

Reply:
Fixed in https://github.com/coin-or/SHOT/pull/7/commits/96673b398bbc1efc35a5ab12c872c10a9ca393c8.